packages/dapp/components/lib/hooks/contracts/use-manager-managed.tsx

```diff
@@ -79,7 +80,7 @@ async function connectManagerContracts(manager: ManagerFacet, provider: NonNulla
   const creditNftCalculator = manager.address;
   const sushiSwapPoolContract = getSushiSwapPoolContract(sushiSwapPool, provider);
 
-  const governanceMarket = getUniswapV2PairContract(await sushiSwapPoolContract.pair(), provider);
+  const governanceMarket = getUniswapV2PairContract(ZERO_ADDRESS, provider);
```

Comment:
@rndquu need some input on this one

Reply:
this was until clarification, not really resseting to a zero address, more input is welcoming